src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.UI/HostConfiguration.cs

```diff
@@ -0,0 +1,139 @@
+namespace MyExtensionsApp;
+
+public static class HostConfiguration
+{
+	// Register services for the application
+	private static void ConfigureServices(IServiceCollection services)
+	{
```

Comment:
Change this to be the same as the other extension methods on IApplicationBuilder

Reply:
actually the pattern I was following was to use the same name as the underlying method that was being called which in this case was Configure instead of Use... if you want to drop the Use then it would make sense to drop the configure

Reply:
Yes, agreed, drop Configure here too